srml/staking/src/lib.rs

```diff
@@ -618,6 +631,13 @@ decl_module! {
 
 		fn deposit_event<T>() = default;
 
+		fn on_initialize() {
+			// Set the start of the first era.
+			if !<CurrentEraStart<T>>::exists() {
+				<CurrentEraStart<T>>::put(T::Time::now());
+			}
```

Comment:
this is needed so that we get the correct value for the first era start. other possibility would be not to reward the first era or having a fixed reward with a parameter.

Also maybe I missed a way to get the actual start of the blockchain ?

Reply:
this code is fine, i think.